Makefile.in

```diff
@@ -172,6 +172,8 @@ default_options_guard.h: default_options.h
 	@printf "/*\n > > > Do not edit this file (default_options_guard.h) < < <\nGenerated from "$^"\nLocal customisation goes in localoptions.h\n*/\n\n" > $@.tmp
 	@./ifndef_wrapper.sh < $^ >> $@.tmp
 	@mv $@.tmp $@
+	@pwd
+	@ls -l $@
```

Comment:
I added these two lines to verify that the default_options_guard.h is indeed created. And that made the Fuzzers pass?... Is that a case of some race condition that gets resolved by spending some time emitting text? Feels like something will come back and bite.
tjk :)

Reply:
_Think_ I finally figured it out, I now have `fuzz/%.o` depending on `$(HEADERS)` in https://github.com/mkj/dropbear/commit/6bc125e6c3c99307c2fb52e59e4f7f4562ac84a1#diff-b48650cabc0f502b52865f525765f2d688198f06e0ffb5456c76fabab7f2bfb2R170